packages/gmp/gmp.6.2.1-5/opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "2.0"
+maintainer: "Lucas Pluvinage <lucas@tarides.com>"
+license: ["LGPL-3.0-only" "LGPL-2.0-only"]
+authors: "Torbjörn Granlund and contributors"
+homepage: "https://github.com/mirage/ocaml-gmp"
+bug-reports: "https://github.com/mirage/ocaml-gmp/issues"
+dev-repo: "git+https://github.com/mirage/ocaml-gmp.git"
+substs: [ "src/build.sh" ]
+build: [
+ [ "dune" "build" "-p" name "-j" jobs ]
+ [ "dune" "runtest" "-p" name "-j" jobs ] {with-test}
+]
+depends: [
+  "ocaml" {>= "4.02.0"}
+  "dune" {>= "2.6"}
+  "conf-m4"
+]
+synopsis: "The GNU Multiple Precision Arithmetic Library"
```

Comment:
```suggestion
depexts: [
  "xz" {os = "macos" & os-distribution = "homebrew"}
]
synopsis: "The GNU Multiple Precision Arithmetic Library"
```

Reply:
Since this is only a problem on macos with homebrew (macports installs xz by default), I this does not warrant a conf package. We can either use a tar.gz or simply add the depext for maxos with homebrew for this release

Reply:
I'm ok with this solution :+1:.